### ISSUE TITLE
Error running constraint.sql on MIMIC-IV version 3.0

### ISSUE THREAD

**Issue by kaveelim** [MIT-LCP/mimic-code#1787]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

I've tried in buide a local mimic on a MacOs system with the MIMIC-IV version 3.0. At the step of running constraint.sql, there's an error...

`psql:mimic-iv/buildmimic/postgres/constraint.sql:207: ERROR:  insert or update on table "diagnoses_icd" violates foreign key constraint "diagnoses_icd_patients_fk"
DETAIL:  Key (subject_id)=(11404231) is not present in table "patients".`

I've checked the shasum256 of the downloaded mimic files. 
Upon inspection, there is no subject_id=11404231 in the patients.csv file but it is present in the diagnoses_icd.csv file.

**Comment by GregSilverman** [comment]
I was also just going to post this issue, but you beat me to it.

**Comment by alistairewj** [comment]
Thanks! Yes I think this was raised in another issue as well. Definitely a bug that we will be investigating in the coming weeks.

**Comment by GregSilverman** [comment]
@alistairewj any progress on this bug?

Thanks!

**Comment by Chessing234** [comment]
yeah this looks like a data integrity issue in the csvs (orphan subject_id in diagnoses_icd), not the constraint script itself — alistairewj flagged it as under investigation. skipping/deferring that fk is the usual workaround until a cleaned release.
